composer.json

```diff
@@ -38,7 +38,7 @@
         "minimaxir/big-list-of-naughty-strings": "dev-master",
         "pear/console_table": "~1.3",
         "phing/phing": "^2.15.0",
-        "php": ">=5.5.9",
+        "php": ">=7.0",
```

Comment:
Is it possible to postpone this change? I'm running locally also the Drupal 6 site in the same time with the new D8. The legacy site is not ready for PHP7 and throws a lot of fatals making it unusable. I think we can develop for PHP7 without this. Take it as temporary.